requirements/doc.txt

```diff
@@ -42,7 +42,7 @@ idna==3.4
     # via requests
 imagesize==1.4.1
     # via sphinx
-importlib-metadata==4.12.0
+importlib-metadata==5.0.0
```

Comment:
No breaking change mentioned in the new version release notes. Removed some deprecated entry points. https://github.com/python/importlib_metadata/blob/main/CHANGES.rst#v500